Update documentation (#121)
1Password/terraform-provider-onepassword: docs/index.md

}
```

> Each of the configuration options can be set using env variables.

Reviewer: ```suggestion
You can set each of the configuration options using environment variables.
```

> That's more secure way to do that as prevents from accidentally pushing sensitive values to the project repo.

Reviewer: ```suggestion
Environment variables are a more secure way to set configuration options because they prevent accidentally pushing sensitive values to the project repo.
```

> - `OP_CONNECT_HOST` - The URL where your 1Password Connect API can be found.

Reviewer: In the Connect documentation, we typically refer to this environment variable as the Connect server's URL/hostname/IP (not the Connect API)... I know it's because we're accessing the API, but I think we should keep it consistent 🙂

- `OP_CONNECT_TOKEN` - A valid token for your 1Password Connect API.
- `OP_SERVICE_ACCOUNT_TOKEN` - A valid 1Password service account token to use with 1Password CLI
- `OP_ACCOUNT` - A valid account's sign-in address or ID to use with 1Password CLI and biometrics unlock.
- `OP_CLI_PATH` - The path to the 1Password CLI binary.

<!-- schema generated by tfplugindocs -->
## Schema

### Optional

> - `account` (String) A valid account's sign-in address or ID to use biometrics unlock. Can also be sourced from OP_ACCOUNT environment variable. Provider will use 1Password CLI if set.
> - `op_cli_path` (String) The path to the 1Password CLI binary. Can also be sourced from OP_CLI_PATH environment variable. Defaults to `op`.
> - `service_account_token` (String) A valid 1Password service account token. Can also be sourced from OP_SERVICE_ACCOUNT_TOKEN environment variable. Provider will use 1Password CLI if set.
> - `token` (String) A valid token for your 1Password Connect API. Can also be sourced from OP_CONNECT_TOKEN environment variable. Provider will use 1Password Connect server if set.
> - `url` (String) The HTTP(S) URL where your 1Password Connect API can be found. Can also be sourced OP_CONNECT_HOST environment variable. Provider will use 1Password Connect server if set.

Reviewer 2: `[nitpick]`
Environment variable names should be wrapped with backticks (`` ` ``) for better highlighting. would it be possible to do that when generating this part of the documentation?

Also, usually for the Connect related configuration we present first the parameter for the URL, followed by the one for the token. Is it possible to make this ordering when generating this part of the documentation?

```suggestion
- `account` (String) A valid account's sign-in address or ID to use biometrics unlock. Can also be sourced from `OP_ACCOUNT` environment variable. Provider will use 1Password CLI if set.
- `op_cli_path` (String) The path to the 1Password CLI binary. Can also be sourced from `OP_CLI_PATH` environment variable. Defaults to `op`.
- `service_account_token` (String) A valid 1Password service account token. Can also be sourced from `OP_SERVICE_ACCOUNT_TOKEN` environment variable. Provider will use 1Password CLI if set.
- `url` (String) The HTTP(S) URL where your 1Password Connect API can be found. Can also be sourced `OP_CONNECT_HOST` environment variable. Provider will use 1Password Connect server if set.
- `token` (String) A valid token for your 1Password Connect API. Can also be sourced from `OP_CONNECT_TOKEN `environment variable. Provider will use 1Password Connect server if set.
```

Author: Regarding backticks - yes, we can wrap env vars.

Regarding the order - no, it's generated automatically in alphabetical order.
